drivers/i2c/stm32f7_i2c.c

```diff
@@ -494,8 +494,26 @@ static int stm32_i2c_xfer(struct udevice *bus, struct i2c_msg *msg,
 			  int nmsgs)
 {
 	struct stm32_i2c_priv *i2c_priv = dev_get_priv(bus);
+	static bool skip = true;
```

Comment:
There are a lot of other devices on the address 0x48: https://i2cdevices.org/addresses/0x48

I'd prefer to store this flag in i2c_priv and bind this workaround to the bus customer uses for connecting SE050, say, declaring a bus address CONFIG...SE050_BUS=0xdeadbeef and checking this address here (i2c_priv->regs == CONFIG...SE050_BUS)

Reply:
sure we can add more stuff to the PR...I didnt think it would be worth the effort but sure, we can put more cushioning around it.

Reply:
then, if we want to make this proper, should I also update OP-TEE to send the connection request package? the current code just assumes that size 260 (max data size) is the connection establishment packet...

Reply:
IMO this patch being so nasty it should just be on a specific factory while the errata is applicable...

Reply:
I would prefer having it in our main tree, as long it can be enabled/disabled via a config or similar. This is just because we expect this logic to be used by multiple factories, and it is impossible to keep track on what is on a factory that should be ported to another one when doing the enablement.

Reply:
ok.